### ISSUE TITLE
Re-write phase details pages

### ISSUE THREAD

**Issue by rglassett** [appacademy/sample-project-proposal#2]
@KletusVandamme used a really nice format in his project repo. It's very close to the format used in lecture--let's make a Google doc that students can copy and emulate.

![image](https://cloud.githubusercontent.com/assets/4405501/6015644/34583e2c-ab2c-11e4-8dbd-74e9f7b7d10a.png)

**Comment by davidrunger** [comment]
I agree that this was a very nice way to outline the project, and worth suggesting/forcing upon students. Do we have an official App Academy Google Drive (and password...) where we can host this document?

I will also float a format that I personally would prefer even more than this spreadsheet style, just a big one-page markdown document detailing all of the phases. Like this: https://gist.github.com/davidrunger/060454209656984f7184#file-one_page_proposal-md
